client/src/main/java/com/paypal/selion/platform/grid/browsercapabilities/ProxyHelper.java

```diff
@@ -12,7 +12,7 @@
  */
 final class ProxyHelper {
 
-    public static final String WARNING_MSG = "Enabling Proxy server settings on %s is known to change the proxy server settings at the machine Level!\n"
+    public static final String WARNING_MSG = "Enabling Proxy server settings on %s is known to change the proxy server settings at the machine Level!%n"
```

Comment:
Why is this changed from `\n` to `%n`?  Also will you please make this line end at 120 characters?